accounts/migrations/0002_alter_customuser_groups_and_more.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0 on 2022-08-20 05:20

```

Comment:
Wondering why this migration file got created? Not seeing any changes to the models?